src/main/java/racingcar/view/InputView.java

```diff
@@ -0,0 +1,38 @@
+package racingcar.view;
+
+import racingcar.view.validator.InputValidator;
+import static racingcar.view.OutputView.*;
+import static racingcar.view.UserMessage.*;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.Scanner;
+
+public class InputView {
+    Scanner sc = new Scanner(System.in);
+    final InputValidator inputValidator = new InputValidator();
+    public List<String> getCarNames() {
+        displayMessage(USER_CAR_NAME_INPUT_PROMPT);
+        String initialInput = sc.nextLine();
+        inputValidator.validateIsBlank(initialInput);
+
+        List<String> parseCarNames = parseCarName(initialInput);
+        inputValidator.validateAvailableName(parseCarNames);
+        return parseCarNames;
+    }
+
+    private List<String> parseCarName (String initialInput) {
+        return Arrays.stream(Arrays.stream(initialInput.split(","))
+                .map(String::trim)
+                .toArray(String[]::new))
+                .toList();
+    }
```

Comment:
한번의 스트림으로도 할 수 있겠네요! `.`을 찍으면 인텔리제이에서 사용가능한 함수를 알려주니 한번 확인해봅시다 😊
```suggestion
        return Arrays.stream(initialInput.split(","))
                .map(String::trim)
                .toList();
    }
```